gems/aws-crt/native/src/api.h

```diff
@@ -42,6 +42,28 @@ AWS_CRT_API void aws_crt_reset_error(void);
 AWS_CRT_API struct aws_event_loop_group *aws_crt_event_loop_group_new(uint16_t max_threads);
 AWS_CRT_API void aws_crt_event_loop_group_release(struct aws_event_loop_group *elg);
 
+/* Auth */
+AWS_CRT_API struct aws_credentials *aws_crt_credentials_new(
+    const char *access_key_id,
+    const char *secret_access_key,
+    const char *session_token,
+    uint64_t expiration_timepoint_seconds);
+AWS_CRT_API void aws_crt_credentials_release(struct aws_credentials *credentials);
+AWS_CRT_API const char *aws_crt_credentials_get_access_key_id(struct aws_credentials *credentials);
+AWS_CRT_API const char *aws_crt_credentials_get_secret_access_key(struct aws_credentials *credentials);
+AWS_CRT_API const char *aws_crt_credentials_get_session_token(struct aws_credentials *credentials);
+AWS_CRT_API uint64_t aws_crt_credentials_get_expiration_timepoint_seconds(struct aws_credentials *credentials);
```

Comment:
trivial: `const struct aws_credentials *credentials` const for all these getters